octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,13 @@
+package org.example.calculator;
+
+public class Calculator {
+
```

Comment:
[Severity: Minor] The main method currently only performs addition with hardcoded values.
Consider allowing user input for the numbers to make the calculator more interactive.
Rationale: User input makes the application more flexible and user-friendly.
Reference: https://www.javatpoint.com/java-scanner-class